src/app/utils.py

```diff
@@ -459,7 +609,9 @@ def check_new_licenses_and_rejected_licenses(inputLicenseText, urlType):
     returns the close matches of license text along with the license issue URL.
     """
     issues = get_rejected_licenses_issues(urlType)
-    issues.extend(get_yet_not_approved_licenses_issues(urlType))
+    notApproved = get_yet_not_approved_licenses_issues(urlType)
+    if isinstance(issues, list):
```

Comment:
Why this test here? Could you not instead ensure that `get_yet_not_approved_licenses_issues` always returns a list or iterable?